clang/test/CodeGen/aie/aie2/aiev2-srs-intrinsics.cpp

```diff
@@ -15,115 +15,469 @@
 // CHECK-NEXT:    [[TMP0:%.*]] = tail call noundef <16 x bfloat> @llvm.aie2.v16accfloat.to.v16bf16(<8 x i64> [[ACC:%.*]])
 // CHECK-NEXT:    ret <16 x bfloat> [[TMP0]]
 //
-v16bfloat16 test_to_v16bfloat16(v16accfloat acc){
-    return to_v16bfloat16(acc);
+v16bfloat16 test_to_v16bfloat16(v16accfloat acc) {
+  return to_v16bfloat16(acc);
 }
 
-// CHECK-LABEL: @_Z14test_ssrs_gen0Dv32_u7__acc32ii(
+// CHECK-LABEL: @_Z9test_ssrsDv32_u7__acc32ii(
 // CHECK-NEXT:  entry:
 // CHECK-NEXT:    [[TMP0:%.*]] = tail call noundef <32 x i8> @llvm.aie2.I256.v32.acc32.srs(<16 x i64> [[ACC:%.*]], i32 [[SHFT:%.*]], i32 [[SIGN:%.*]])
```

Comment:
nit: Do we want to keep the flat I256 next to the structured v32.acc32 in the llvm intrinsic name? I find it untidy.